applications/IgaApplication/tests/cpp_tests/test_support_laplacian_condition.cpp

```diff
@@ -0,0 +1,99 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License: BSD License
+//           Kratos default license: kratos/license.txt
+//
+
+// Project includes
+#include "containers/model.h"
+#include "testing/testing.h"
+
+#include "test_creation_utility.h"
+
+#include "custom_conditions/support_laplacian_condition.h"
+#include "includes/convection_diffusion_settings.h"
+
+namespace Kratos
+{
+namespace Testing
+{
+    ///@name Type Definitions
+    ///@{
+
+    typedef std::size_t SizeType;
+    typedef std::size_t IndexType;
+
+    ///@}
+    ///@name Operations
+    ///@{
+
+    typename Condition::Pointer GetSupportLaplacianCondition(
+        ModelPart& rModelPart, SizeType PolynomialDegree, IntegrationPoint<3> IntegrationPoint)
+    {
+        // Set the element properties
+        auto p_elem_prop = rModelPart.CreateNewProperties(0);
+        p_elem_prop->SetValue(PENALTY_FACTOR, -1); // Penalty-free formulation
+
+        auto p_quadrature_point = TestCreationUtility::GetQuadraturePointGeometryOnCurve(
+            rModelPart, PolynomialDegree, IntegrationPoint);
+        p_quadrature_point->SetValue(TEMPERATURE, 0.24);
+
+        return Kratos::make_intrusive<SupportLaplacianCondition>(1, p_quadrature_point, p_elem_prop);
+    }
```

Comment:
By doing it like this you're defining the `GetSupportLaplacianCondition` for the entire `Kratos::Testing` namespace. If you plant to use it in other places, you should move it to an auxiliary file. If not, I'd put it into an anonymous namespace so it is only accessible from this file (just in case).